doc: Add note about IP retrieval in 'README.md' (#694)
ionos-cloud/terraform-provider-ionoscloud: README.md

  ## Migrating from the ProfitBricks provider

  Please see the [Documentation](docs/index.md#migrating-from-the-profitbricks-provider) on how to migrate from the ProfitBricks provider.

  ## Frequently Asked Questions
    
  ### Are there any Terraform resources that can show us the configured DHCP IP for a specific NIC?
  Please check out this [module](https://github.com/ionos-cloud/terraform-ionoscloud-kube-lan-ip).

>   ### How do we find out the IP for the added NIC on a K8s nodepool?

Reviewer: ```suggestion
  ### How can I find out the IP for the added NIC on a K8s nodepool?
```

Author: @mimihalescu done in 3c81ef20870c9ea12e17464a12b4d05fb317c37b
